Packs/EDL/Integrations/EDL/EDL_description.md

```diff
@@ -1,36 +1,13 @@
 ## How to Access the Generic Export Indicators Service
-<~XSOAR>
```

Comment:
Seems to me that the removal of XSOAR / XSIAM tags from this file was done automatically.
Please review it and revert if these removals are un intended.